docs/current_docs/api/clients-sdk.mdx

```diff
@@ -0,0 +1,418 @@
+
+Try the Go program by executing the command below from the project directory:
+
+```shell
+dagger run go run multibuild/main.go
+```
+
+The `dagger run` command executes the specified command in a Dagger session and displays live progress. The Go program builds the application for each OS/architecture combination and writes the build results to the host. You will see the build process run four times, once for each combination. Note that the builds are happening concurrently, because the builds do not depend on eachother.
+
+Use the `tree` command to see the build artifacts on the host, as shown below:
+
+```shell
+tree build
+build
+├── 1.22
+│   ├── darwin
+│   │   ├── amd64
+│   │   │   └── hello
+│   │   └── arm64
+│   │       └── hello
+│   └── linux
+│       ├── amd64
+│       │   └── hello
+│       └── arm64
+│           └── hello
+└── 1.23
+    ├── darwin
+    │   ├── amd64
+    │   │   └── hello
+    │   └── arm64
+    │       └── hello
+    └── linux
+        ├── amd64
+        │   └── hello
+        └── arm64
+            └── hello
+```
+</TabItem>
+<TabItem value="Python">
+
+:::note
+The Dagger Python SDK requires [Python 3.10 or later](https://docs.python.org/3/using/index.html).
+:::
+
+Using a [virtual environment](https://packaging.python.org/en/latest/tutorials/installing-packages/#creating-virtual-environments) is recommended. Create a virtual environment for your project:
+
+```shell
+python3 -m venv .venv && source .venv/bin/activate
+```
+
+Install the Dagger Python SDK in your project's virtual environment using [PyPI](https://pypi.org/project/dagger-io/) or [Conda](https://anaconda.org/conda-forge/dagger-io) (from the [conda-forge](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) channel):
+
+```shell
+# using pypi
+pip install dagger-io
+
+# using conda
+conda install dagger-io
+```
+
+This example demonstrates how to test a Python application against multiple Python versions using the Python SDK.
+
+Clone an example project:
+
+```shell
+git clone --branch 0.101.0 https://github.com/tiangolo/fastapi
+cd fastapi
+```
+
+Create a new file named `test.py` in the project directory and add the following code to it.
+
+```python file=./snippets/clients-sdk/python/test.py
+```
+
+This Python program imports the Dagger SDK and defines an asynchronous function named `test()`. This `test()` function creates a Dagger client, which provides an interface to the Dagger API. It also defines the test matrix, consisting of Python versions `3.7` to `3.11` and iterates over this matrix, downloading a Python container image for each specified version and testing the source application in that version.
+
+Run the Python program by executing the command below from the project directory:
+
+```shell
+dagger run python test.py
```

Comment:
Using `uv run` allows uv to manage the Python installation, virtual environment creation, and dependencies installation, without having to activate the environment too:
```suggestion
dagger run uv run test.py
```

For more on running scripts with uv: https://docs.astral.sh/uv/guides/scripts/

For example, in the simplest cases, uv allows you to define the dependency in the script itself, so all you need is that one file.